File: app/src/main/java/com/roman/kubik/spivanyklicejista/presentation/splash/SplashActivity.java
package com.roman.kubik.spivanyklicejista.presentation.splash;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;

import com.roman.kubik.spivanyklicejista.presentation.Navigate;

/**
 * Launcher activity
 * Created by kubik on 1/14/18.
 */

public class SplashActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Navigate.toMainActivity(this);
    }
}

File: app/src/main/java/com/roman/kubik/spivanyklicejista/presentation/splash/SplashActivity.kt
package com.roman.kubik.spivanyklicejista.presentation.splash

import android.os.Bundle
import android.support.v7.app.AppCompatActivity

import com.roman.kubik.spivanyklicejista.data.database.DatabaseCopyHelper
import com.roman.kubik.spivanyklicejista.presentation.Navigate

import java.io.IOException

/**
 * Launcher activity
 * Created by kubik on 1/14/18.
 */

class SplashActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        try {
            DatabaseCopyHelper(this).createDataBase()
            Navigate.toMainActivity(this)
        } catch (ignored: IOException) {
        }
        finish()
    }
}
